projects/ADuCM3029_demo_cn0428_cn0429/src/ADuCM3029_demo_cn0428_cn0429.cpp

```diff
@@ -0,0 +1,1307 @@
+/**
```

Comment:
For all your source/header files please use the licence header comment found here: https://gitlab.analog.com/Platformation/sw-tools/blob/master/documentation/coding-style.md

It was created to fit the required code length. As you might see, the current one that you are using is larger than the allowed limit and Astyle is messing it up.

Reply:
I can't access this link either. I used this file that Mircea linked to me: https://github.com/analogdevicesinc/arduino/blob/master/LICENSE.txt
Does that mean we need to have that file fixed as well?

Reply:
I think that this is an older file that was included just to get the repo licensed. For code development we prefer using the standard one found in the documentation link above. You should have now access to it.  Sorry for the misunderstanding.